src/Register.php

```diff
@@ -0,0 +1,45 @@
+<?php namespace Leean\Elements;
+
+use Leean\Elements\Options;
+
+/**
+ * Class to provide predefined ACF elements.
+ */
+class Register
+{
+	/**
+	 * The location for the elements.
+	 *
+	 * @var array
+	 */
+	private static $_location = [];
+
+	/**
+	 * Registered elements.
+	 *
+	 * @var array
+	 */
+	private static $_elements = [];
+
+	/**
+	 * Init. Takes the location and the elements to register in that location.
+	 *
+	 * @param array $location Array suitable for ACF location parameter.
+	 * @param arrat $elements Array of element names to show in this location.
+	 */
+	public static function init( $location, $elements ) {
+		self::$_location = $location;
```

Comment:
Yes, agreed. But in this case we can set a default.
We could also add a filter to the 'location' => key in the acf config if we wanted to make it overridable.